upgrade/upd_2.5.10-to-2.5.11/index.php

```
@@ -736,20 +798,20 @@ public function apply_templatesadmin()
      *
      * @return bool
      */
-    public function check_zapsmarty()
+    public function check_zapsmarty(): bool
     {
-        return !file_exists('../class/smarty/smarty.class.php');
+        return !file_exists(XOOPS_ROOT_PATH . '/class/smarty/smarty.class.php');
     }
```

PHPMD (2.15.0)

[error] The method check_zapsmarty is not named in camelCase. (undefined)

(CamelCaseMethodName)